app/views/shared/_sul_header.html.erb

```diff
@@ -8,9 +8,9 @@
     <% end %>
     <nav aria-label="header menu">
       <ul class="list-unstyled d-flex gap-4 m-0">
-        <% if current_user.sso_user? %>
+        <% if current_user.sso_user? || current_user.library_id_user? %>
           <li>
-            <%= link_to t('.logout', sunetid: current_user.sunetid), logout_path(referrer: root_path) %>
+            <%= link_to t('.logout', sunetid: current_user.sunetid || current_user.library_id), logout_path(referrer: root_path) %>
```

Comment:
I don't think we want to send library id users to the root path. That page is kinda a dead-end for them:

<img width="790" alt="Screenshot 2024-04-03 at 08 56 45" src="https://github.com/sul-dlss/sul-requests/assets/111218/1b32e624-c9b8-4df6-9330-6fcdf7867f73">

Reply:
Do you have a suggestion of somewhere else? Right now we are sending sunet there as well but I am guessing because of the redirect in warden they aren't actually getting there.

Reply:
Yeah, that's tricky. For admin users, the root path makes sense.. but if it's a normal patron's placing a request, maybe it makes sense to send them back to the first request page (where they can choose to login again, maybe as a different user?)